libraries/AP_HAL_ChibiOS/hwdef/MttrCubeOrange/defaults.parm

```diff
@@ -189,7 +193,7 @@ SERIAL3_PROTOCOL	5
 SERIAL4_BAUD	115
 SERIAL4_PROTOCOL	100
 SERIAL5_BAUD	57
-SERIAL5_PROTOCOL -1
+SERIAL5_PROTOCOL 2
```

Comment:
For my understanding. What does 2 mean in this instance?

Reply:
it means MAVLink2

Reply:
1 = MAVLink 1
2 = MAVLink 2
5 = GPS